NetKAN/VenStockRevamp.netkan

```diff
@@ -13,5 +13,15 @@
     "comment"      : ".version file currently has accurate mod version but incorrect KSP version",
 	"resources" : {
         "homepage"     : "http://forum.kerbalspaceprogram.com/threads/92764"
-    }
+    },
+        "x_netkan_override" : [
+        {
+            "version" : "v1.2",
```

Comment:
This would have the match the tag of the most recent release which is [`v1.8.1`](https://github.com/VenVen/Stock-Revamp/releases/tag/v1.8.1).